godtools/App/Share/Domain/UseCases/GetOptInOnboardingTutorialAvailableUseCase/GetOptInOnboardingTutorialAvailableUseCase.swift

```diff
@@ -21,4 +22,10 @@ class GetOptInOnboardingTutorialAvailableUseCase {
         
         return deviceLanguage.isEnglish
     }
+    
+    func getOptInOnboardingTutorialIsAvailable() -> AnyPublisher<Bool, Never> {
+        
+        return Just(getOptInOnboardingTutorialIsAvailable())
```

Comment:
Oh cool!  I was wondering how you convert a single value into a publisher when there's not a stream of data.  That's handy.